extension/src/utils/ApiInterface.js

```diff
@@ -2,6 +2,23 @@ const API_HOST = "localhost:3000";
 const GET_REQUEST = "GET";
 const POST_REQUEST = "POST";
 const STUDENT_NAME = "student_name";
+const stickers = [ 
+    {
+        id: "golden_star", 
+        price: 50,
```

Comment:
pump the prices higher please, coins are awarded in the increments of hundreds and thousands.

For the purpose of demo, please adjust these into higher ranges (e.g. 1000, 10000, 1000000)